ajax/libs/just-animate/package.json

```diff
@@ -11,18 +11,19 @@
     "javascript",
     "animation",
     "timeline",
-    "sequence"
+    "sequence",
+    "waapi"
   ],
   "license": "MIT",
-  "homepage": "https://just-animate.github.io/",
+  "homepage": "http://just-animate.com/",
   "repository": {
     "type": "git",
     "url": "https://github.com/just-animate/just-animate"
   },
   "npmName": "just-animate",
   "npmFileMap": [
     {
-      "basePath": "browser",
+      "basePath": "dist",
       "files": [
         "**/!(*.ts)"
```

Comment:
@notoriousb1t I think we can replace `"**/!(*.ts)"` with `"**/!(index.js)"`here.